server/model/yaml/encoding.go

```diff
@@ -50,6 +51,13 @@ func Decode(contents []byte) (File, error) {
 			return File{}, fmt.Errorf("cannot decode datastore: %w", err)
 		}
 		f.Spec = dataStore
+	case FileTypeConfig:
```

Comment:
We now support config resource files

Reply:
@xoscar can you add a test for this structure on `encoding_test.go`, please?